RulerDetectionPlugin/src/RulerSettingsDialog.cpp

```diff
@@ -29,6 +29,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include <qfiledialog.h>
 #include <qsizepolicy.h>
 #include <qlayout.h>
+#include <QtUiTools\quiloader.h>
```

Comment:
remove - that results in a compile error
gerneral remark: you can/should include class names rather than header files (e.g. `#include <QFileDialog>`).